contracts/implementations/MultipleMinter.sol

```diff
@@ -27,7 +28,9 @@ contract MultipleMinter is
     SyntheticDeviceInternal
 {
     bytes32 private constant MINT_VEHICLE_SD_TYPEHASH =
-        keccak256("MintVehicleAndSdSign(uint256 integrationNode)");
+        keccak256(
```

Comment:
For synthetics we have a lot of control over the device wallet, so we could do this.

But we could get away with—is there a reason this doesn’t include the owner’s address? Then I’d feel good about using the user nonce, as for aftermarket.

Reply:
I guess for synthetics they can only ever get minted once. Although I guess you could re-use a signature from a burned one, so maybe that’s not true.